pyproject.toml

```diff
@@ -1,2 +1,2 @@
 [build-system]
-requires = ["setuptools", "wheel", "numpy", "cython"]
+requires = ["setuptools >= 40.8.0", "wheel", "numpy", "cython"]
```

Comment:
I honestly don't remember why 40.8.0 was the settled-on version, but I'm not sure that I care to titrate it back to see if we can get to 30.3.0 or some earlier one. At least it's not a regression to leave it as it is.

@oesteban Good to merge as-is?

Reply:
Very good!